glide.yaml

```diff
@@ -24,3 +24,6 @@ import:
 - package: github.com/kubernetes-incubator/apiserver-builder
   version: ^1.9.0-alpha.3
 - package: sigs.k8s.io/cluster-api
+  version: master
```

Comment:
Do we really need to explicitly specify master for the version? Isn't that the default?

Reply:
I don't know what's going on but specifying master appears to actually be dead wrong, it was giving me an Aug 17th commit for testify somehow. Removing it and I get the proper master version. Updating.